docs/quickstart.md

```diff
@@ -0,0 +1,31 @@
+# Quickstart
+
+## About
+
+This document provides a quick set up for an up-to-date version of the Permanent Identifier Service. For a quickstart on YOURLS, see https://yourls.org/docs/guide/install. For a quickstart as a contributor, see [Contributor](./contributor/) for the set up of a development environment.
```

Comment:
You refer to PID above so you could probably say that here, unless it has a different meaning here.